### DISCUSSION TITLE
Dreadnought Robotics Update thread

### DISCUSSION THREAD

**Discussion by dreadnoughtroboticsvitcc** [dreadnoughtrobotics/dreadnoughtrobotics.github.io#27] [Ideas]
Thread for the upcoming DNT website's update

**Comment by PratikGarai** [comment]
Lightbox for gallery ; 
https://www.w3schools.com/howto/howto_js_lightbox.asp

**Comment by PratikGarai** [comment]
Petition to remove "About VIT" from home [age of DNT website

**Comment by dreadnoughtroboticsvitcc** [comment]
Phaser.js fpv, torpedo control game inside dedicated pages of bots
